optimade/server/main_index.py

```diff
@@ -133,6 +133,7 @@ def add_optional_versioned_base_urls(app: FastAPI):
 
 @app.on_event("startup")
 async def startup_event():
+    CONFIG.is_index = True
```

Comment:
It is not clear to me why this line of code is here. 
If this line is run every time the server is started, would the effect not be the same as setting the default value to true ?

Reply:
Yes, for the index server this is desired behavior. The intention is that any user-supplied value for this field will be ignored.